src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,23 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
-    public int getUserScore(String[] records, String email) {
-        return 0;
+    public int getUserScore(String[] records, String email) throws UserNotFoundException {
+        int userScore = -1;
+        for (String row : records) {
+            String[] dividedRow = row.split(":");
+            if (dividedRow[0].contains(email)) {
+                userScore = Integer.parseInt(dividedRow[1]);
+                break;
+            }
+        }
+
+        if (userScore < 0) {
```

Comment:
just throw exception if you've iterated whole loop ( that means no matches found) in this case useScore would be redundant